plane/src/controller/mod.rs

```diff
@@ -150,14 +154,14 @@ impl ControllerServer {
             .route("/c/:cluster/drone-socket", get(handle_drone_socket))
             .route("/c/:cluster/proxy-socket", get(handle_proxy_socket))
             .route("/dns-socket", get(handle_dns_socket))
-            .route("/c/:cluster/connect", post(handle_connect))
+            .route("/c/connect", post(handle_connect))
             .route("/c/:cluster/d/:drone/drain", post(handle_drain))
             .route(
-                "/c/:cluster/b/:backend/soft-terminate",
+                "/b/:backend/soft-terminate",
                 post(terminate::handle_soft_terminate),
             )
             .route(
-                "/c/:cluster/b/:backend/hard-terminate",
+                "/b/:backend/hard-terminate",
                 post(terminate::handle_hard_terminate),
```

Comment:
This is one of the more impactful changes of this PR; a bunch of paths no longer refer to clusters.